packages/google-cloud-storage-control/README.rst

```diff
@@ -1,9 +1,9 @@
-Python Client for Cloud Storage
-===============================
+Python Client for Storage Control API
```

Comment:
This file is automatically generated using information in `.repo-metadata.json` however you can generate the file locally. In a branch of `google-cloud-python`, run the following commands

**Pull the latest docker image**

- `docker pull gcr.io/cloud-devrel-public-resources/owlbot-python-mono-repo:latest`

**create a directory which will signal the post processor to regenerate templated files**

- `mkdir -p owl-bot-staging/google-cloud-storage-control`

**Run the post processor locally**

- `docker run --user $(id -u):$(id -g) --rm -v $(pwd):/repo -w /repo gcr.io/cloud-devrel-public-resources/owlbot-python-mono-repo:latest`

**Restore autogenerated samples which are erroneously removed as a result of using this process**

- `git checkout packages/google-cloud-storage-control/samples`

Reply:
@parthea Thank you for the instructions! Ran post processor and successfully regenerated the README file.

Reply:
Is there a way to trigger owlbot post processor, if needed? I don't seem to have access to any label triggers. Appreciate any help to merge the PR :)

Reply:
Done